docs/src/content/docs/book/send.mdx

```diff
@@ -92,13 +94,31 @@ send(SendParameters{
     // bounce is set to true by default
     to: address,
     value: value,
-    mode: SendIgnoreErrors, // don't stop in case of errors
+    mode: SendIgnoreErrors, // skip the message in case of errors
     code: init.code,
     data: init.data,
     body: "Hello, World!".asComment(), // not necessary, can be omitted
 });
 ```
 
+<p/><Badge text="Available since Tact 1.6 (not released yet)" variant="tip" size="medium"/><p/>
+
+For cheaper on-chain deployments prefer using the [`deploy(){:tact}`](/ref/core-common#deploy) function instead. It computes the address of the contract based on its initial code and data and efficiently composes the resulting message:
+
+```tact
+let init: StateInit = initOf SecondContract(arg1, arg2);
+deploy(DeployParameters{
+    // bounce is set to true by default
+    mode: SendIgnoreErrors, // skip the message in case of errors
+    value: ton("1"), // a whole Toncoin
+    body: "Hello, World!".asComment(), // not necessary, can be omitted
+    init: StateInit{
+        code: init.code,
+        data: init.data,
+    },
```

Comment:
this is a roundabout way of doing
```suggestion
    init,
```

and it's even better to inline `init`:

```
...
init: initOf SecondContract(arg1, arg2)
```

and move it to the top since it would make it clearer what contract we are deploying